pkg/utils/validator/clusterresourceplacement.go

```diff
@@ -30,6 +39,17 @@ func ValidateClusterResourcePlacementAlpha(clusterResourcePlacement *fleetv1alph
 				allErr = append(allErr, fmt.Errorf("the labelSelector in resource selector %+v is invalid: %w", selector, err))
 			}
 		}
+		if ResourceInformer != nil {
+			gvk := schema.GroupVersionKind{
+				Group:   selector.Group,
+				Version: selector.Version,
+				Kind:    selector.Kind,
+			}
+			// TODO: Ensure gvk created from resource selector is valid.
```

Comment:
it's more about differentiate the case that the cache is not populated vs not a cluster scope resource